examples/platform/s3-cache.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_s3_bucket" "platform_runner_cache" {
+  bucket = "platform-runner-cache"
+}
+
+resource "aws_s3_bucket_policy" "platform_runner_cache_bucket_policy" {
+  bucket = aws_s3_bucket.platform_runner_cache.id
+  policy = data.aws_iam_policy_document.platform_runner_cache_policy.json
+}
+
+data "aws_iam_policy_document" "platform_runner_cache_policy" {
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::854439639386:role/platform/platform-runner-role"]
```

Comment:
How can I reference this ARN from a different module?

Reply:
Not sure if you can, unless you just use a variable and reference that instead. That's probably what I'd do (if I'm understanding your question correctly). There are some examples of doing that here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document

Reply:
Cheers, the S3 and IAM resources appear to be working now using a variable, but I won't merge this until the resources are actually working with sccache correctly in case more changes are needed.